### PR TITLE
op-guide: update ansible and offline deployment

### PR DESCRIPTION
**Pull request by lilin90**
@LinuxGit @CaitinChen PTAL
Via: https://github.com/pingcap/docs-cn/pull/682 & https://github.com/pingcap/docs-cn/pull/685

### CODE REVIEW

File: op-guide/offline-ansible-deployment.md [hunk 1]
```diff
@@ -19,41 +19,62 @@ Before you start, make sure that you have:
...
-    Use the following method to install Ansible online on the download machine installed with the CentOS 7 system. Installing using the EPEL source automatically installs the related Ansible dependencies (such as `Jinja2==2.7.2 MarkupSafe==0.11`). After Ansible is installed, you can view the version using `ansible --version`.
+    Use the following method to install Ansible online on the download machine installed with the CentOS 7 system. After Ansible is installed, you can view the version using `ansible --version`.
 
     ```bash
     # yum install epel-release
     # yum install ansible curl
     # ansible --version
-
-     ansible 2.4.2.0
+      ansible 2.4.2.0
```
Comment (Hoverbear) [inline op-guide/offline-ansible-deployment.md]: I think this should also be `2.5.0`?

Comment (LinuxGit) [reply op-guide/offline-ansible-deployment.md]: yep

File: op-guide/offline-ansible-deployment.md [hunk 2]
```diff
@@ -19,41 +19,62 @@ Before you start, make sure that you have:
...
-    # tar -xzvf ansible-2.4.2-rpms.el7.tar.gz
+    # tar -xzvf pip-rpms.el7.tar.gz
+    # cd pip-rpms.el7
+    # chmod u+x install_pip.sh
+    # ./install_pip.sh
+    ```
+
+    After the installation is finished, you can use `pip -V` to check whether it is successfully installed:
+
+    ```
```
Comment (CaitinChen) [inline op-guide/offline-ansible-deployment.md]: Add "bash" after "```"

File: op-guide/offline-ansible-deployment.md [hunk 3]
```diff
@@ -19,41 +19,62 @@ Before you start, make sure that you have:
...
-    Use the following method to install Ansible online on the download machine installed with the CentOS 7 system. Installing using the EPEL source automatically installs the related Ansible dependencies (such as `Jinja2==2.7.2 MarkupSafe==0.11`). After Ansible is installed, you can view the version using `ansible --version`.
+    Use the following method to install Ansible online on the download machine installed with the CentOS 7 system. After Ansible is installed, you can view the version using `ansible --version`.
 
     ```bash
     # yum install epel-release
     # yum install ansible curl
     # ansible --version
-
-     ansible 2.4.2.0
+      ansible 2.5.0
```
Comment (CaitinChen) [inline op-guide/offline-ansible-deployment.md]: ansible 2.4.2.0

Comment (lilin90) [reply op-guide/offline-ansible-deployment.md]: This is updated in another PR: https://github.com/pingcap/docs-cn/pull/685. So just keep it.

**Pull request merged**